splinepy/io/gismo.py

```diff
@@ -305,18 +343,22 @@ def export(
         f.write(file_content)
 
 
-def load(fname):
+def load(fname, get_options=True):
```

Comment:
```suggestion
def load(fname, load_options=True):
```
do you need this? why not just always load options if it is specified?

Reply:
because it requires that a tuple is returned every time. If you are ok with that I can change that